src/cityreader/cityreader.py

```diff
@@ -60,12 +74,26 @@ def cityreader(cities=[]):
 
 # TODO Get latitude and longitude values from the user
 
+lat_one = input("Give me your latidude value one")
```

Comment:
maybe use something like `input("Give me your longitude value two \n ") `  so that, input is sufficiently separated from the string.